cmd/agent/install_script.sh

```diff
@@ -310,6 +345,18 @@ if [ "$OS" = "RedHat" ]; then
       dnf_flag="--best"
     fi
 
+    if [ -n "$agent_minor_version" ]; then
+        # Example: datadog-agent-7.20.2-1
+        pkg_pattern="$agent_major_version\.${agent_minor_version%.}(\.[[:digit:]]+){0,1}(-[[:digit:]])?"
```

Comment:
Have you tested this with RCs? I think it should work fine but we should check it

Reply:
I haven't. how do I test this with RC? are there a different apt/yum url?

Reply:
Left you a Slack message!

Reply:
tested and works with RCs. Following command worked in Centos8, Fedora33, Ubuntu20, Debian9, Opensuse15

`DD_AGENT_MAJOR_VERSION=7 DD_AGENT_MINOR_VERSION=28.0~rc.7 REPO_URL=datad0g.com DD_AGENT_DIST_CHANNEL=beta DD_API_KEY=xxx bash -c "$(curl -H 'Cache-control: no-cache' -L https://raw.githubusercontent.com/DataDog/datadog-agent/ian.bucad/opt_min_version/cmd/agent/install_script.sh)"`